src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -211,7 +161,7 @@ protected int doWork() {
         for (int i = 0; i < windowsByGc.length; ++i) {
```

Comment:
This is also true in the original code...feel free to leave it despite the warning.